### DISCUSSION TITLE
Whitelist or Allow list in prowler 3.0

### DISCUSSION THREAD

**Discussion by NMuee** [prowler-cloud/prowler#2235] [Q&A] [answered]
Hi prowler team, 

Apologise if i missed it, is there a whitelist / allowlist function in prowler 3.0 similar to what prowler 2 has? 
Thank you.

**Comment by jfagoagas** [comment, marked as answer]
Hi @NMuee, we do! You can check here https://docs.prowler.cloud/en/latest/tutorials/allowlist/ the tutorial to use the Allowlist feature with Prowler v3.

Thanks for using Prowler 🚀
